randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1526,23 +1500,21 @@ Extra - map_name: rm_a5b07
   * Normal Door to Facility Storage Tower Route South/Door to Facility Storage Tower East; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door
   * Extra - instance_id: 135756
   > Dock to Facility Storage Clogged Pipe South
-      Any of the following:
-          Power Grip or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
+      Underwater Power Grip Wall
   > Dock to Spiky Space Jump Trial Entrance
       Trivial
 
 > Dock to Facility Storage Clogged Pipe South; Heals? False
   * Layers: default
   * Open Passage to Facility Storage Clogged Pipe South/Dock to Facility Storage Tower East
   > Dock to Facility Storage Tower Route North
-      Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
+      All of the following:
+          Gravity Suit and Speed Booster and Disabled Entrance Rando
+          Any of the following:
+              Shinesparking Tricks (Beginner)
+              All of the following:
+                  # Jump to break the blocks
+                  Movement (Beginner) and Space Jump Wall
```

Comment:
This space jump wall doesnt really fit with "jump". There's spider, IBJ and WJ in there, all of them negate the speedboost effect without shinesparking.
This should probably be either space jump or HJ. Don't think you can reach that without any jump items.
HJ may or may not need power grip or walljumping too, in the case where its possible to break the speed blocks, but not get to the top.